### ISSUE TITLE
Some docstringsneed to be turned to raw strings

### ISSUE THREAD

**Issue by dliessi** [wbsoft/frescobaldi#645]
Reported by @proski in https://github.com/wbsoft/frescobaldi/issues/637#issuecomment-78126940.

**Comment by dliessi** [comment]
@proski: could you please give some more details?

**Comment by proski** [comment]
Suppose a function has a docstring:
`"""Insert \rest"""`
It's readable in the program text. However, if somebody uses the `help` command, they would see:
`Insert ^Mest`
Also, pylint would spell check "est" rather than "rest". The fix is to use this raw docstring:
`r"""Insert \rest"""`
Note that `\u` is not allowed in raw docstrings as long as we want Python 2 compatibility.

**Comment by PeterBjuhr** [comment]
If I understand this correctly,

`r"""Insert \rest"""`

is preferable over

`"""Insert \\rest"""`

??

(Perhaps with the exception for `\\u` in Python 2.)

I think we use both constructs at the moment...

**Comment by proski** [comment]
That's correct. The former is better because it's more readable in the source file.
